src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -10,27 +10,55 @@ class Tribe__Events__Importer__File_Importer_Events extends Tribe__Events__Impor
 	protected function match_existing_post( array $record ) {
 		$start_date = $this->get_event_start_date( $record );
 		$end_date   = $this->get_event_end_date( $record );
+		$all_day    = $this->get_boolean_value_by_key( $record, 'event_all_day' );
+
+		// Base query - only the meta query will be different
 		$query_args = array(
 			'post_type'      => Tribe__Events__Main::POSTTYPE,
 			'post_title'     => $this->get_value_by_key( $record, 'event_name' ),
-			'meta_query'     => array(
+			'fields'         => 'ids',
+			'posts_per_page' => 1,
+		);
+
+		// When trying to find matches for all day events, the comparison should only be against the date
+		// component only since a) the time is irrelevant and b) the time may have been adjusted to match
+		// the eod cutoff setting
+		if ( in_array( $all_day, array( 'yes', 'true', '1' ) ) ) {
```

Comment:
OK, done, and after a further look, no meed to modify _is_all_day()_ in this case as it turns out we don't support `1` as a possible truthy value for that field (maybe that's what you were hinting at?).